src/types/types.ts

```diff
@@ -45,3 +45,10 @@ export interface ICourseDetailsResponseData{
   completedChapterLength: number,
   chaptersLength: number,
 }
+
+export interface IMessageResponseData{
```

Comment:
Why do you create another IMessageResponseData if you already have MessageProps in componentType.tsx?

Reply:
fixed